TombEngine/Objects/TR4/tr4_objects.cpp

```diff
@@ -249,7 +249,7 @@ namespace TEN::Entities
 			obj->pivotLength = 50;
 			obj->radius = 102;
 			obj->intelligent = true;
-			obj->ZoneType = ZoneType::Human;
```

Comment:
It's not necessary to comment it here. When animations / states will be added, then we can change zone type again. Otherwise, it's original behaviour and it doesn't require `TODO` comment here.